### ISSUE TITLE
[Question] CreateConversationAsync throws error: Incorrect conversation creation parameters

### ISSUE THREAD

**Issue by benmetheny** [Microsoft/BotBuilder#3061]
Following the example for a bot in Teams Channel [here](https://msdn.microsoft.com/en-us/microsoft-teams/botsinchannels#creating-new-channel-conversation) throws an HttpOperationException with message "Incorrect conversation creation parameters". I'm pulling the ChannelId directly from my current context TeamsChannelData and I've tried setting all sorts of different parameters in the ConversationParameters object. I do see the Note above the example code referenced above about not being able to initiate 1:Many conversations but I'm unsure if that means that the example code does not actually work (yet).

**Comment by nwhitmont** [comment]
@benmetheny I don't understand your question. What are you trying to accomplish here exactly? Also, are you using .NET or Node SDK? Please post your bot code so we can better assist you.

**Comment by benmetheny** [comment]
I'm using .NET trying to create a new Conversation in Teams Channel.

**Comment by benmetheny** [comment]
I got this working - it seems that Teams expects the parameters in a different format than the TeamsChannelData class serializes to. I was looking at the Activity.ChannelData that Teams was sending and noticed the difference. There are just two top level properties that contain duplicate info in the existing info, substituting the class below with TeamsChannelId property filled correctly causes the example code I linked to to work.

     public class TeamsChannelDataEx : TeamsChannelData {
        [JsonProperty("teamsChannelId")]
        public string TeamsChannelId { get; set; }

        [JsonProperty("teamsTeamId")]
        public string TeamsTeamId { get; set; }
    }
